### PR TITLE
update AutoConnect-ASCSubscriptions

### PR DESCRIPTION
**Pull request by swiftsolves-msft**
updated existing playbook AutoConnect-ASCSubscriptions.

1. Leverages MSI now instead of App Registration

2. Updates connector to use Bi-directional sync with Azure Defender

3. updated readme with information with changes.

Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Playbooks/AutoConnect-ASCSubscriptions/readme.md [hunk 1]
```diff
@@ -21,9 +22,9 @@ Documentation references:
 <li><a href="https://docs.microsoft.com/azure/governance/management-groups/overview" target="_blank" rel="noopener">Learn more about Azure Management Groups</a></li>
 </ul>
 </li>
-<li>Azure Active Directory registered application, assigned with RBAC roles
+<li>Logic App as a Managed Service identity - MSI, assigned with RBAC roles
 <ul>
-<li><a href="https://docs.microsoft.com/azure/active-directory/develop/app-objects-and-service-principals" target="_blank" rel="noopener">Learn more about applications in Azure Active Directory.</a></li>
+<li><a href="https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview" target="_blank" rel="noopener">Learn more about Managed Service Identities in Azure Active Directory.</a></li>
```
Comment (sarah-yo) [inline Playbooks/AutoConnect-ASCSubscriptions/readme.md:27]: please remove the "en-us" from the URL

**Review by sarah-yo — CHANGES_REQUESTED** [review]
Please remove the localisation in the URL (marked in the file) so the document validation doesn't fail, thanks!

**Comment by lior-tamir** [comment]
@swiftsolves-msft Thanks for the great update!!
Why did we get the resource group/subscription as a parameter instead of the ARM template function?

**Comment by swiftsolves-msft** [comment]
> @swiftsolves-msft Thanks for the great update!!
> Why did we get the resource group/subscription as a parameter instead of the ARM template function?

Thanks Lior apologies I ahve restored those deploy funcs and removed the params.

**Review by lior-tamir — APPROVED** [review]

**Review by sreedharande — APPROVED** [review]

**Pull request merged**